guides/common/modules/proc_applying-configurations.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Applying configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases Satellite restart is required.
+
+It is also a good practice to setup a monitoring before applying any change as it will allow you to evaluate effect of the change.
+Our testing environment might be too far from what you will see although we are trying hard to mimic real world environment.
+
+If you have changed some systemd service file, you need to notify systemd daemon to reload the configuration:
+
+----
+# systemctl daemon-reload
+----
+
+Note this will still need to reload the service to do the actual change. It is best to restart whole Satellite:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} service restart
+----
+
+If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml` you need to rerun installer to apply the changes:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-installer}
+----
+
+Also you might want to rerun installer with some new option, so you can:
+
+[options="nowrap" subs="attributes,quotes"]
+----
+# {foreman-installer} _new options_
+----
+
+After any change, please run this quick {Project} health-check:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} health check
+Running ForemanMaintain::Scenario::FilteredScenario
+================================================================================
+Check number of fact names in database:                               [OK]
+--------------------------------------------------------------------------------
+Check whether all services are running:                               [OK]
+--------------------------------------------------------------------------------
+Check whether all services are running using the ping call:           [OK]
+--------------------------------------------------------------------------------
+Check for paused tasks:                                               [OK]
+--------------------------------------------------------------------------------
+Check whether system is self-registered or not:                       [OK]
+--------------------------------------------------------------------------------
```

Comment:
```suggestion
```

We generally don't include console output. Instead, you can say "Verify that all services are running:
command ...
"

Reply:
Removed, thank you